docs/deploy/ios.rst

```diff
@@ -148,18 +148,28 @@ Customize the App
 
 We can customize the iOS app in several ways.
```

Comment:
Could we add an end-to-end tutorial for:
1. Adding a model variant (e.g. adding OpenHermes -- hence no need to compile model)
2. Adding a new model (e.g. adding RedPajama -- hence need to compile model)

We could just give some concise steps, then ask user to refer to our [convert_weight](https://llm.mlc.ai/docs/compilation/convert_weights.html) and [compile_model](https://llm.mlc.ai/docs/compilation/compile_models.html) pages correspondingly. 

For web-llm, these correspond to:
1. https://llm.mlc.ai/docs/deploy/javascript.html#bring-your-own-model-variant
2. https://llm.mlc.ai/docs/deploy/javascript.html#bring-your-own-model-library

Reply:
Sounds great, I will work on that in the meantime